src/controllers/update/update.manage.trusts.review.legal.entities.controller.ts

```diff
@@ -1,31 +1,66 @@
 import { NextFunction, Request, Response } from 'express';
 
-import { logger } from '../../utils/logger';
+import { Session } from '@companieshouse/node-session-handler';
+
 import {
-  UPDATE_MANAGE_TRUSTS_INDIVIDUALS_OR_ENTITIES_INVOLVED_URL,
+  UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_URL,
   UPDATE_MANAGE_TRUSTS_REVIEW_INDIVIDUALS_URL,
   UPDATE_MANAGE_TRUSTS_REVIEW_LEGAL_ENTITIES_PAGE,
+  UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_LEGAL_ENTITY_URL,
 } from '../../config';
+import { logger } from '../../utils/logger';
+import { getApplicationData, setExtraData } from '../../utils/application.data';
+import { saveAndContinue } from '../../utils/save.and.continue';
+import { getTrustInReview, setTrusteesAsReviewed } from '../../utils/update/review_trusts';
+import { ApplicationData } from '../../model';
+import { TrusteeType } from '../../model/trustee.type.model';
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
+    const appData = getApplicationData(req.session);
+    const trustInReview = getTrustInReview(appData);
+    const legalEntities = trustInReview?.CORPORATES;
+
+    if (!legalEntities) {
+      throw new Error('Failed to render Manage Trusts Review legal entities page. No legal entities in session');
```

Comment:
could just skip to next phase of review?

Reply:
It's a scenario we _really_ shouldn't end up in, so I don't think it _really_ matters. The middleware should catch it and redirect, but if that somehow doesn't happen we'll get this error.